packages/database/lib/schema.ts

```diff
@@ -275,3 +297,31 @@ export const candles = pgTable('candles', {
 }, table => ({
   pk: primaryKey(table.marketAcct, table.candleDuration, table.timestamp)
 }));
+
+export const comments = pgTable('comments', {
+  // Need this as we reference this for response and nesting
+  commentId: bigint('comment_id', {mode: 'bigint'}).notNull(),
+  // Generated when comment is created
+  commentorAcct: pubkey('commentor_acct').notNull(),
+  proposalAcct: pubkey('proposal_acct').notNull(),
+  // This will be the body content of the comment
+  content: text('content').notNull(),
+  // Use only if its a responding comment in a chain, we should constrain this so
+  // it references only commentIds which have respondingCommentId with NULL
+  respondingCommentId: bigint('responding_comment_id', {mode: 'bigint'}).references(() => comments.commentId),
+  createdAt: timestamp('created_at').notNull().default(sql`now()`)
+}, table => ({
+  pk: primaryKey(table.commentId, table.commentorAcct, table.proposalAcct),
+  first: unique('commentor_content').on(table.commentorAcct, table.content)
+}));
+
+export const reactions = pgTable('reactions', {
+  reactorAcct: pubkey('reactor_acct').notNull(),
+  proposalAcct: pubkey('proposal_acct').references(() => proposals.proposalAcct).notNull(),
+  reaction: pgEnum('reaction', Reactions).notNull(),
+  updatedAt: timestamp('updated_at').notNull(),
+}, table => ({
+  // Note: we maybe should do a unique reactorAcct + proposalAcct, but
+  // at the very least should unique reactorAcct + proposalAcct + reaction
+  pk: primaryKey(table.proposalAcct, table.reaction, table.reactorAcct)
```

Comment:
Should we restrict reactions to one per person per proposal, or one per type or?